### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [waynerv/waynerv.github.io#35] [Announcements]
# about/

你好，欢迎来到 Shall We Code！我是 Waynerv，一名后端工程师，我会在博客中分享我的所思所得，涉及的领域包括但不限于：Pytho……

https://waynerv.com/about/

**Comment by oliverdding** [comment]
博文帮助了我许多，感谢Waynerv！

**Comment by Asongguo** [comment]
文章很棒！

**Comment by yunzhan2014** [comment]
感谢作者，收益良多。

**Comment by coolCodeStudy** [comment]
Waynerv 你好，请问方便分享一下你的background嘛？

**Comment by nikuleo** [comment]
感谢作者，订阅了

**Comment by HHN0609** [comment]
rebase写的很好！
